pkgs/development/libraries/speechd/default.nix

```diff
@@ -1,38 +1,88 @@
-{ fetchurl, lib, stdenv, intltool, libtool, pkgconfig, glib, dotconf, libsndfile
-, libao, python3Packages
-, withEspeak ? false, espeak
+{ stdenv, lib, pkgconfig, fetchurl, wrapPython
+, intltool, itstool, libtool, texinfo, autoreconfHook
+, glib, dotconf, libsndfile
+, pyxdg
+, withLibao ? true, libao
+, withPulse ? false, libpulseaudio
+, withAlsa ? false, alsaLib
+, withOss ? false
+, withFlite ? true, flite
+, withFestival ? false # TODO: , festival-freebsoft-utils
```

Comment:
I would comment out the whole attribute, since it does nothing.

Reply:
Idem for Ivona.
Then also in the configureFlags the optional arguments.
Done.